scipy/sparse/linalg/_eigen/lobpcg/lobpcg.py

```diff
@@ -16,12 +16,15 @@
 .. [3] A. V. Knyazev's C and MATLAB implementations:
        https://github.com/lobpcg/blopex
 """
+# Author: Andrew Knyazev <Andrew.Knyazev@ucdenver.edu>
+#         Robert Cimrman <https://github.com/rc>
```

Comment:
You can use `Robert Cimrman <cimrman3@ntc.zcu.cz>` here (the github page is OK too).

Reply:
Done